From a11e5c2d325b1c3fc63ec428597459b3813e8d09 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 8 Jun 2018 10:28:43 +0800
Subject: [PATCH] arm64: dts: rockchip: Set pmic-reset-func to 1 for rk3326
 linux

As the config reset registers only for rk817 and rk809.

Change-Id: Id8e8ae3b816c783df92510083c36156595d230ae
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts
index 98d3081a8a50..3ca25895939f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts
@@ -362,7 +362,7 @@
 		clock-output-names = "rk808-clkout1", "rk808-clkout2";
 		//fb-inner-reg-idxs = <2>;
 		/* 1: rst regs (default in codes), 0: rst the pmic */
-		pmic-reset-func = <0>;
+		pmic-reset-func = <1>;
 
 		vcc1-supply = <&vccsys>;
 		vcc2-supply = <&vccsys>;
-- 
2.35.3

